docs/src/architecture/09_architecture_decisions/0003-app-ver-control.md

```diff
@@ -0,0 +1,35 @@
+---
+    title: 0003 Hermes runtime extensions api version control
+    adr:
+        author: Alex Pozhylenkov <alex.pozhylenkov@iohk.io>
+        created: 16-Feb-2024
+        status:  draft
+---
+
+## Context
+
+Hermes should be a robust and reliable platform for Hermes applications to be run on.
+So it has to have a defined boundaries of which application would be executed and which not.
+
+## Assumptions
+
+* Hermes runtime extensions api could not be stable and evolve during the development process.
+
+## Decision
+
+As a part of the metadata of the Hermes application, provide a `api_version` field.
```

Comment:
We are too immature to decide a version policy or even how it should be implemented.

For now, lets just make this:

"There is an acknowledged need for version control of the APIs in a mature Hermes state, however at this stage we will not define such a policy until the core hermes engine and libraries have matured enough to make an informed choice."

Or something like that.